src/operations/common/params/gitHubPatterns.ts

```diff
@@ -0,0 +1,12 @@
+
+export const GitHubNameRegExp = {
```

Comment:
These look familiar.

Reply:
I don't know what to do with this comment?
 are they duplicated somewhere?